config/test_cases/test_branch_origin_integration.yml

```diff
@@ -6,4 +6,5 @@ extensions:
       title: "run integration tests"
       repository: "origin"
       script: |-
-        JUNIT_REPORT='true' make test -o check -k
\ No newline at end of file
+        OS_BUILD_ENV_DOCKER_ARGS="-v /tmp:/openshifttmp " OS_BUILD_ENV_PRESERVE=_output/local hack/env JUNIT_REPORT='true' make test-tools test-integration -o check -k
+        JUNIT_REPORT='true' make test-tools test-end-to-end -o check -k
```

Comment:
We don't need the `test-tools` target down here, unless you expect there to be a difference between inside and outside of `hack/env`?